src/containers/Board/SymbolSearch/SymbolSearch.css

```diff
@@ -10,6 +10,14 @@
 
 .react-autosuggest__suggestion {
   width: 25%;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+
+  /* Adds a hyphen where the word breaks */
+  -webkit-hyphens: auto;
+  -ms-hyphens: auto;
```

Comment:
react-scripts uses ‘AutoPrefixer’ package. I think if you remove these we should be fine